taichi/backends/vulkan/runtime.cpp

```diff
@@ -560,6 +562,19 @@ void VkRuntime::launch_kernel(KernelHandle handle, RuntimeContext *host_ctx) {
     }
   }
 
+  // If we have accumulated some work but does not require sync
+  // and if the accumulated cmdlist has been pending for some time
+  // launch the cmdlist to start processing.
+  if (current_cmdlist_) {
+    constexpr uint64_t max_pending_time = 2000;  // 3000us = 2ms
```

Comment:
```suggestion
    constexpr uint64_t max_pending_time = 2000;  // 2000us = 2ms
```

Reply:
btw, do we need to elaborate here why 2ms?

Reply:
It's just a value I just decided on. (A queue submission is on the scale of 0.1ms) Doesn't seem to cause too much overhead & pretty good at overlapping CPU work with GPU especially for the JIT scenario.